### PR TITLE
#594 Add PBO import column aliasing, disable PBO data file import

### PR DESCRIPTION
**Pull request by oroztocil**
Related issue #594

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ Projekt aktuálně nemá vytvořené testy, ale budeme rádi když s nimi pomů
 Frontend dle [Angular Style Guide](https://angular.io/guide/styleguide).
 Code Quality dle automaticky kontrolovaných ESLint pravidel (pre-commit hook a kontrola každého pull-request/push)
 
+### Prettier formatter
+
+Kód lze nechat automaticky zformátovat pomocí příkazu:
+
+```bash
+prettier . --write
```
Comment (prusovak2) [inline CONTRIBUTING.md:46]: 👍 
udelej issue na sjednoceni stylu  mezi serverem a klinetem

File: server/src/worker/import/cityvizor/importer.ts [hunk 2]
```diff
@@ -10,6 +10,8 @@ import {PostprocessingTransformer} from '../postprocessing-transformer';
 import {CityvizorFileType} from './cityvizor-file-type';
 
 export async function importCityvizor(options: Import.Options) {
+  logger.log(`Starting import: ${JSON.stringify(options)}}`);
```
Comment (prusovak2) [inline server/src/worker/import/cityvizor/importer.ts:13]: 😓

**Review by prusovak2 — APPROVED** [review]
🚀

File: CONTRIBUTING.md (hunk 1 above)
Comment (oroztocil) [reply CONTRIBUTING.md:46]: https://github.com/cityvizor/cityvizor/issues/597

**Pull request merged**